src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1338,9 +1338,21 @@ class WorkspaceService(protected val userInfo: UserInfo, dataSource: DataSource,
               containerDAO.authDAO.saveGroup(group, txn)
               level -> group
             }
+
+            def addOwnerToOwnerIntersectionGroup(ownerIntersectRef: RawlsGroupRef) = {
+              val ownerIntersect = containerDAO.authDAO.loadGroup(ownerIntersectRef, txn).getOrElse(throw new RawlsException("Unable to add owner to group"))
+              gcsDAO.addMemberToGoogleGroup(ownerIntersect, Left(RawlsUser(userInfo)))
+              containerDAO.authDAO.saveGroup(ownerIntersect.copy(users = Set(RawlsUser(userInfo))), txn)
+            }
```

Comment:
2 problems with this function: 1) it should probably be in the gcs dao setupWorkspace call since that is where the rest of the logic is. 2) addMemberToGoogleGroup returns a future the result of which is lost.